e2e/tests/oai/003-core-network.sh

```diff
@@ -38,7 +38,7 @@ function _wait_for_pfcp_session {
 
     info "waiting for PFCP session to be established"
     timeout=600
-    until kubectl logs "$(kubectl get pods -n oai-core --kubeconfig "$kubeconfig" -l workload.nephio.org/oai=upf -o jsonpath='{.items[*].metadata.name}')" -n oai-core -c upf-edge --kubeconfig "$kubeconfig" | grep -q 'Received SX HEARTBEAT REQUEST'; do
+    until kubectl logs -l workload.nephio.org/oai=upf -n oai-core -c upf-edge --kubeconfig "$kubeconfig" --tail -1 | grep -q 'Received SX HEARTBEAT REQUEST'; do
```

Comment:
@electrocucaracha I think it is better to have at least 50 lines rather than only 1.

Reply:
The value is `-1` , this is the default value when no selector is provided.  https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands

Reply:
True my bad I missed the `-`.